type_docx.go

```diff
@@ -208,6 +211,40 @@ type DocxBlocUndefined struct{}
 // DocxBlocQuoteContainer 引用容器 Block
 type DocxBlocQuoteContainer struct{}
 
+// DocxBlockOKR OKR 信息
+type DocxBlockOKR struct {
+	OKRID               string                      `json:"okr_id,omitempty"`                // OKR ID
+	PeriodDisplayStatus *DocxOKRPeriodDisplayStatus `json:"period_display_status,omitempty"` // 周期的状态
```

Comment:
```suggestion
	PeriodDisplayStatus DocxOKRPeriodDisplayStatus `json:"period_display_status,omitempty"` // 周期的状态
```

这个是 string，不是结构体，不需要加 *